src/data/markdown/docs/40 xk6-disruptor/01 Get started/03 Installation.md

```diff
@@ -15,7 +15,7 @@ The quickest way to get started is to [download a release binary from GitHub](ht
 
 You can also use [xk6](https://github.com/grafana/xk6) to build a k6 binary.
 
-To find out more about how to use xk6 or what it is, check out this guide - [Build a k6 binary with extensions](/extensions/guides/build-a-k6-binary-with-extensions/).
+To find out more about how to use xk6 or what it is, check out either [Build a k6 binary using Go](/extensions/guides/build-a-k6-binary-using-go/) or [Build a k6 binary using Docker](/extensions/guides/build-a-k6-binary-using-docker/).
```

Comment:
This is a bit confusing. The section is called `Build from source` and should only include instructions to build from source. Maybe we need a section similar to what you have in extensions, `Support for Docker`, that points people toward building it with Docker. Otherwise, maybe "repeat" instructions in another section, on how to build it with Docker, on this same page?